app/models/openscap_result.rb

```diff
@@ -0,0 +1,68 @@
+require 'openscap'
+require 'openscap/ds/arf'
+require 'openscap/xccdf/benchmark'
```

Comment:
Does openscap have a loader that requires the files it needs?  It seems if openscap adds features, we'll need to keep this list of requires up to date.